### PR TITLE
[open62541] Update version to 1.3.4

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

File: ports/open62541/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "open62541",
-  "version": "1.2.3",
+  "version": "1.3.4",
   "port-version": 1,
```
Comment (JonLiu1993) [inline ports/open62541/vcpkg.json:4]:
```suggestion
```

File: ports/open62541/portfile.cmake [hunk 2]
```diff
@@ -28,6 +25,7 @@ vcpkg_cmake_configure(
     OPTIONS
         ${FEATURE_OPTIONS}
         -DOPEN62541_VERSION=${VERSION}
+        -DUA_FORCE_WERROR=OFF
```
Comment (JonLiu1993) [inline ports/open62541/portfile.cmake:28]: @autoantwort, Thanks for your PR, the option is UA_FORCE_WERROR? I searched and did not find this option.

**Comment by JonLiu1993** [comment]
Please get [failure logs here](https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/83810/artifacts?artifactName=failure%20logs%20for%20x86-windows&api-version=7.0&%24format=zip).

**Comment by JonLiu1993** [comment]
Pinging @autoantwort for response. Is work still being done for this PR?

**Comment by autoantwort** [comment]
I think I can close this temporarily

**Pull request closed without merging**